web/src/features/moreCast2/components/MoreCast2DataGrid.tsx

```diff
@@ -1,21 +1,27 @@
 import React from 'react'
 import makeStyles from '@mui/styles/makeStyles'
-import {
-  DataGrid,
-  GridColDef,
-  GridValueFormatterParams,
-  GridValueGetterParams,
-  GridValueSetterParams
-} from '@mui/x-data-grid'
-import { DateTime } from 'luxon'
-import { ModelChoice } from 'api/moreCast2API'
+import { DataGrid, GridColDef, GridEventListener } from '@mui/x-data-grid'
+import { ModelType } from 'api/moreCast2API'
 import { MoreCast2ForecastRow } from 'features/moreCast2/interfaces'
-import { LinearProgress } from '@mui/material'
+import { LinearProgress, Menu, MenuItem } from '@mui/material'
 import { useSelector } from 'react-redux'
 import { selectMorecast2TableLoading } from 'app/rootReducer'
+import ApplyToColumnMenu from 'features/moreCast2/components/ApplyToColumnMenu'
+import { isEqual } from 'lodash'
+import { DateRange } from 'components/dateRangePicker/types'
+import { FireCenterStation } from 'api/fbaAPI'
+import { MORECAST2_FIELDS } from 'features/moreCast2/components/MoreCast2Field'
 
 interface MoreCast2DataGridProps {
```

Comment:
I think you can trim down the MoreCast2DataGridProps to:
  - rows
   - clickedColDef
  - updateColumnWithModel
  - setClickedColDef